README.md

```diff
@@ -209,7 +210,7 @@ $ make
 ```
 
 If you also want to try the example Java benchmarks, you must build them
-as a separate step:
+as a separate step. Additionally, be sure to have compiled Krun with the java option on Step 3.
```

Comment:
How about "Additionally, be sure to have compiled Krun with Java support as described in Step 3"?